garage/tf/algos/npo.py

```diff
@@ -323,18 +344,16 @@ def _build_policy_loss(self, i):
                         surr_clip = lr_clip * advantages * i.valid_var
                     else:
                         surr_clip = lr_clip * adv_valid
-                    surr_obj = tf.minimum(
-                        surr_vanilla, surr_clip, name="surr_obj")
+                    obj = tf.minimum(surr_vanilla, surr_clip, name="surr_obj")
                 else:
                     raise NotImplementedError("Unknown PGLoss")
```

Comment:
can you test this case?

Reply:
Tested. Also tested "Unknown KLConstraint"